.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,79 @@
+name: Docker
+
+on:
+  push:
+    # Publish `master` as Docker `latest` image.
+    branches:
+      - master
+
+    # Publish `v1.2.3` tags as releases.
+    tags:
+      - v*
```

Comment:
I just realized, I don't use a v prefix for versioning in bot-kt, can you fix this?

Reply:
Yes, of course

Reply:
Appears to stop this from running on PRs...